itest/rust/src/builtin_tests/convert_test.rs

```diff
@@ -233,3 +233,510 @@ fn custom_convert_error_from_variant() {
         format!("{:?}", i64::MAX)
     );
 }
+
+// Ensure valid conversions of Godot arrays to Rust vectors work.
+#[itest]
+fn vec_from_godot_valid() {
+    let valid_array_i64 = array! {
+        2_i64,
+        5_i64,
+    };
```

Comment:
```suggestion
    let valid_array_i64 = array![2_i64, 5_i64];
```
Should generally use `array![...]` syntax, and for short expressions like here, can use a single line.

Many occurrences like this can be simplified.